app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="text-center">Sign up</h2>
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= f.error_notification %>
+  <div class="form-inputs m-4">
+    <%= f.input :name,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "name" }%>
+    <%= f.input :email,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "email" }%>
+    <%= f.input :password,
+                required: true,
+                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                input_html: { autocomplete: "new-password" } %>
+    <%= f.input :password_confirmation,
+                required: true,
+                input_html: { autocomplete: "new-password" } %>
+  </div>
+  <div class="form-actions ms-4">
+    <%= f.button :submit, "Sign up", data:{turbo:false}, class:"btn btn-primary"%>
+  </div>
+<% end %>
+<div class="ms-4">
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] We are required to maintain the design. Kindly redesign your form pages to match the given design.

  Your Implementation | Behance Design
  :---:|:---:
  ![opwe](https://user-images.githubusercontent.com/54848363/214599740-9bfa8cdb-389c-44ef-ae2a-bc4ae100bbc4.jpg) | ![siow](https://user-images.githubusercontent.com/54848363/214599749-409a4e97-d1ec-4e73-baf5-7faacdf166b3.jpg)